src/Spice86/ViewModels/MainWindowViewModel.cs

```diff
@@ -184,7 +184,7 @@ public sealed partial class MainWindowViewModel : ObservableObject, IGui, IDispo
         IVideoBufferViewModel videoBuffer = new VideoBufferViewModel(videoCard, scale, bufferWidth, bufferHeight, address, VideoBuffers.Count, isPrimaryDisplay);
         Dispatcher.UIThread.Post(
             () => {
-                if(!VideoBuffers.Any(x => x.Address == videoBuffer.Address)) {
+                if(VideoBuffers.All(x => x.Address != videoBuffer.Address)) {
```

Comment:
Any should be faster than All, since it stops at the first item where the condition is not true ?

Reply:
`Any` also stops at the 1st entry that matches the condition doesn't it?

So I think it't just a syntactic difference between "if not any of the buffers have this address" and "if all of the buffers don't have this address".

Reply:
My bad. The performance is the same.